contracts/governance/AxelarGovernance.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { Upgradable } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/upgradable/Upgradable.sol';
+import { IAxelarGateway } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/interfaces/IAxelarGateway.sol';
+import { IAxelarGovernance } from '../interfaces/IAxelarGovernance.sol';
+import { AdminMultisigBase } from './AdminMultisigBase.sol';
+
+contract AxelarGovernance is Upgradable, AdminMultisigBase, IAxelarGovernance {
```

Comment:
And we need a separate contract that doesn't check for governance for the separate operator role. Could technically reuse the same contract and governance address to `0`, but I'd prefer to be more explicit